src/app/component/projects/pages/harvest/harvest.component.scss

```diff
@@ -0,0 +1,4 @@
+// Set width of prime ng steps so that they fill screen
+.ui-steps .ui-steps-item {
+  width: 16.66%; // 100 / 6
```

Comment:
this seems to be hacky.

Can we get the items inside to scale to their box? Flexbocx should do this